### PR TITLE
Add docs and fix rate limiting for websockets of dYdX

### PR DESCRIPTION
**Pull request by davidsblom**
# Pull Request

Add docs and fix rate limiting for websockets of dYdX

## Type of change

Delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How has this change been tested?

Live example

### CODE REVIEW

File: nautilus_trader/adapters/dydx/websocket/client.py [hunk 1]
```diff
@@ -105,13 +105,28 @@ def is_disconnected(self) -> bool:
...
+
+        Parameters
+        ----------
+        item : tuple[str, str]
+            Topic name.
+
+        Returns
+        -------
+        bool
+            Whether the client is already subscribed to this topic.
```
Comment (cjdsellers) [inline nautilus_trader/adapters/dydx/websocket/client.py:129]: I think it's best to add only additional info relative to the return type for the description, in this case it's been covered in the main description already.

File: nautilus_trader/adapters/dydx/websocket/client.py [hunk 2]
```diff
@@ -147,10 +157,26 @@ async def connect(self) -> None:
...
 
     def _msg_handler(self, raw: bytes) -> None:
+        """
+        Handle pushed websocket messages.
+
+        Parameters
+        ----------
+        raw : bytes
+            The received message in bytes.
+        """
```
Comment (cjdsellers) [inline nautilus_trader/adapters/dydx/websocket/client.py:167]: Small point, but we tend to leave a single blank line at the end of the docstring. The `docformatter` lint may just handle this when its turned back on.

**Review by cjdsellers — APPROVED** [review]

**Comment by cjdsellers** [comment]
Thanks @davidsblom :pray:

File: nautilus_trader/adapters/dydx/websocket/client.py (hunk 2 above)
Comment (davidsblom) [reply nautilus_trader/adapters/dydx/websocket/client.py:167]: Thanks for formatting the code already.

File: nautilus_trader/adapters/dydx/websocket/client.py (hunk 1 above)
Comment (davidsblom) [reply nautilus_trader/adapters/dydx/websocket/client.py:129]: Makes sense. Thanks!

**Pull request merged**